lib/secureAccessMethods.js

```diff
@@ -174,10 +174,28 @@ function createSecure(user, options) {
         });
 }
 
+/**
+ * allows a protected method to be called
+ * @param  {string} method   protected method to call
+ * @param  {object} options  options to call function with
+ * @return {Promise}         Promise that the called function returns
```

Comment:
cory recently pointed out that `return` actually isnt a thing.  `returns` is the proper.  who knew. not important, but if you wanna.